projects/demo/src/modules/components/input-year/examples/1/index.html

```diff
@@ -1,10 +1,15 @@
-<tui-input-year
-    [formControl]="control"
-    [tuiTextfieldCleaner]="true"
->
-    Choose a year
+<tui-textfield #textfield>
     <input
-        placeholder="Not 2022 please"
-        tuiTextfieldLegacy
+        placeholder="Choose a year"
+        tuiInputYear
+        [(ngModel)]="value"
     />
-</tui-input-year>
+
+    <ng-template tuiTextfieldDropdown>
+        <tui-calendar-year
+            [initialItem]="value || initial"
```

Comment:
We need a way to not set it explicitly. It should be 
a) current year if value is not set
b) value year if it's set
c) initialItem if it is explicitly set